Program.cs

```diff
@@ -29,6 +31,10 @@ public static void Main(string[] args)
             Views views = new Views(blockChain);
             Menus menus = new Menus(views);
             views.TopView.Add(menus.MenuBar);
+            if (blockIndex != 0)
+            {
+                Dialogs.IndexSearchAction(views, blockIndex.ToString());    
+            }
```

Comment:
Then, what view do you expect when running with `-i 0` options to look genesis block? If you wanted to check whether the option was given, How about replace `blockIndex`'s type with `long?`?.

Reply:
@moreal I've applied it!